install-dev/data/xml/quick_access.xml

```diff
@@ -6,7 +6,7 @@
   </fields>
   <entities>
     <quick_access id="Orders" new_window="0">
-      <link>index.php?controller=AdminOrders</link>
+      <link>index.php/sell/orders</link>
```

Comment:
don't forget the upgrade script please?

Reply:
Nice reminder :)

https://github.com/PrestaShop/autoupgrade/pull/457